File: 백준/Bronze/8393. 합/합.java
import java.io.*;

public class Main {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int num1=Integer.parseInt(br.readLine());
        int sum=0;

        for(int i=1; i<=num1; i++) {
            sum += i;
        }

        System.out.println(sum);

    }

}

File: 백준/Bronze/8393. 합/합.kt
import java.io.*
import java.util.*

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))

    var num = Integer.parseInt(br.readLine())

    println(addAllNumber(num))
}

fun addAllNumber(n1 : Int) : Int{
    var sum = 0
    for(i in 1.. n1) {
        sum+=i
    }

    return sum
}

